modules/quant_api/quant_api.admin.inc

```diff
@@ -12,47 +12,58 @@
  */
 function quant_api_settings() {
   if (!variable_get('quant_api_ssl_verify', TRUE)) {
-    $form['markup'] = array(
+    $form['warning'] = array(
       '#type' => 'markup',
       '#prefix' => '<div class="messages error">',
       '#suffix' => '</div>',
-      '#markup' => '<b>DANGER ZONE</b> SSL verification is disabled for Quant API connections, it is recommended that you address server configuration rather than disabling SSL verification.'
+      '#markup' => '<strong>DANGER ZONE:</strong> SSL verification is disabled for Quant API connections. It is <strong>highly recommended</strong> that you update your server configuration to handle SSL rather than disabling SSL verification.'
     );
   }
 
+  $css_class = 'status';
+  if (empty(variable_get('quant_api_customer')) || empty(variable_get('quant_api_project')) || empty(variable_get('quant_api_token'))) {
+    $css_class = 'warning';
+  }
+  $form['help'] = array(
+    '#type' => 'markup',
+    '#prefix' => '<div class="messages ' . $css_class . '">',
+    '#suffix' => '</div>',
+    '#markup' => 'You can find your API information in your Quant dashboard. In the dashboard, first choose your organization and project. Then, go to the <code>Integrations</code> page and the API information is shown in the <code>Integration details</code> section.'
```

Comment:
Does it make sense to include a links to the docs site for this information as well?